### DISCUSSION TITLE
Get a file icon from a webview?

### DISCUSSION THREAD

**Discussion by mkestler-rtp** [microsoft/vscode-discussions#91] [Extension Development QnA] [answered]
Is there any way to get the themed file icon from a webview?  For instance the "TS" icon of a typescript file.  It is trivial with a treeview using ThemeIcon and the URI, but I haven't found a way to acquire one from a webview.  Even if I could get a URI to one and send it over to the webview, I'd be fine.

Thanks!

**Comment by burkeholland** [comment, marked as answer]
Per @misolori, webviews are sandboxed off from the other API's. As far as I know there is no API call you can make to get a file path for a specific icon.
